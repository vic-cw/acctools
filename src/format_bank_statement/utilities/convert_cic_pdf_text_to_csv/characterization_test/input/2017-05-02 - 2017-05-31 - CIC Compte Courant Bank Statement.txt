                                                                       Crédit Industriel et Commercial

                                                                                                                CIC PARIS SAINT MARTIN

                                                                                                          224 RUE SAINT MARTIN 75010 PARIS
                                                   ¤      u0820 81 23 45 (Service 0,12 /min + prix appel) FAX 01 44 16 81 39
                                                                                                                                           12345@cic.fr                                         BIC : CMCIFRPP

                                                                                                                                                                                                                34567

                                                                                                                                                                                                                12345

                                                                                                                          M  JACK  SMITH

                                                                                                                          1  FREEDOM   STREET

                                                                                                                          75005    PERPETTE

...........................................................................................................................V..O...T.R..E...C..O..N..S..E..I.L..L..E..R..:..M...S..é..b..a.s..t.i.e.n...M..A..R..T..I.N......................

                                             RELEVE ET INFORMATIONS BANCAIRES                                                                                                                                Agence 12345

                                             31 mai 2017

                                                   C/C CONTRAT PERSONNEL GLOBAL N° 00012345678 en euros (GD)

                                                   TITULAIRE(S) : M JACK SMITH

                                             Date        Date valeur                    Opération                                                                                               Débit EUROS  Crédit EUROS

                                                                                        SOLDE CREDITEUR AU 02/05/2017                                                                                        1.234,56                                                                                        Pour tout appel en vue d'obtenir la bonne exécution d'un contrat ou le traitement

                                             02/05/2017  02/05/2017                    RETRAIT DAB 2704 PLACE                                                                                   50,00

                                             04/05/2017  04/05/2017                    PAIEMENT CB    0305 12345678910                                                                          244,00

KV.20170601.012345.6789.1011.1213 X 0 2                                                MERCHANT       CARTE 00123456

                                             05/05/2017  05/05/2017                    ONE LINE EXPENSE                                                                                         695,52

                                             (GE) : protégé par la Garantie de l'Etat                                                                                                                        <<Suite au verso>>

                                             (GD) : protégé par la Garantie des Dépôts

                                             Remarque : Le livret réglementé relève de la garantie d'Etat et le(s) livret(s) ordinaire(s) de dépassement associé(s) de la garantie des dépôts.

                                             www.garantiedesdepots.fr

                                                                                                                                                                                                             Page 1

                                               CREDIT INDUSTRIEL ET COMMERCIAL (CIC) - Banque régie par les articles L.511-1 et suivants du Code Monétaire et Financier - SA au capital de 608 439 888 

                                         6,  avenue de Provence 75009 Paris - Adresse postale : 75452 Paris cedex 09 - tél 01 45 96 96 96 - swift CMCIFRPP - www.cic.fr - 542 016 381 RCS Paris - TVA intracommunautaire : FR34542016381

                                                                                        M le Médiateur du CIC : 63 chemin Antoine Pardon 69160 Tassin la demi lune - www.lemediateur-cic.fr

                                                                       Pour les opérations effectuées en sa qualité d'intermédiaire en opérations d'assurance : ORIAS 07 025 723 (www.orias.fr)
                                         RELEVE ET INFORMATIONS BANCAIRES                                                                                                                                    Agence 12345

                                         31 mai 2017

                                               C/C CONTRAT PERSONNEL GLOBAL N° 00012345678                                                                       en  euros    (GD)

                                               TITULAIRE(S) : M JACK SMITH

                                         Date        Date valeur                    Opération                                                                                       Débit         EUROS      Crédit EUROS

                                         24/05/2017  24/05/2017                    INCOMING FUNDS                                                                                                            1.234,56

                                                                                   REASON      /A01B23456789101

                                         31/05/2017  31/05/2017                    PAIEMENT PSC 2905 NICE                                                                                         1,10

KV.20170601.012345.6789.1011.1213 X 0 V                                            MARCHAND         CARTE 00123456

                                         Réf : 005                                  SOLDE CREDITEUR AU 31/05/2017                                                                                            1.234,56

                                           SITUATION DE VOS CREDITS en euros au 31 mai 2017

                                         Numéro                                     Libellé du compte                                                                         Montant emprunté               Solde (*)

                                         00012345678               ALLURE LIBRE                                                                                          EUR                      +0,00    +0,00

                                         00012345679               ETALIS                                                                                                EUR                      +0,00  +0,00

                                         (*) Solde correspondant à l'amortissement normal du prêt

                                         AUTORISATIONS DE DECOUVERTS

                                         N° de compte                               Libellé                                                                                                                  Montant euros

                                         00012345678               DECOUVERT C. PERS. GLOBAL TAEG 15,12% AU 26/04/2017                                                                                       5.000.000,00

                                         (GE) : protégé par la Garantie de l'Etat

                                         (GD) : protégé par la Garantie des Dépôts

                                         Remarque : Le livret réglementé relève de la garantie d'Etat et le(s) livret(s) ordinaire(s) de dépassement associé(s)      de  la garantie des dépôts.

                                         www.garantiedesdepots.fr

                                                                                                                                                                                                             Page 2
