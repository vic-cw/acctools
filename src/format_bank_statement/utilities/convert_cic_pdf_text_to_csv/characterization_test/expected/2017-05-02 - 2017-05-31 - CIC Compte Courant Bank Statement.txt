2017-05-02,2017-05-02,"RETRAIT DAB 2704 PLACE","-50,00"
2017-05-04,2017-05-04,"PAIEMENT CB    0305 12345678910 MERCHANT       CARTE 00123456","-244,00"
2017-05-05,2017-05-05,"ONE LINE EXPENSE","-695,52"
2017-05-24,2017-05-24,"INCOMING FUNDS REASON      /A01B23456789101","1.234,56"
2017-05-31,2017-05-31,"PAIEMENT PSC 2905 NICE MARCHAND         CARTE 00123456","-1,10"
